tests/snappi_tests/multidut/test_sys_trial.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from tests.common.helpers.assertions import pytest_require, pytest_assert                           # noqa: F401
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts, fanout_graph_facts_multidut    # noqa: F401
+from tests.common.snappi_tests.snappi_fixtures import snappi_api_serv_ip, snappi_api_serv_port, \
+    snappi_api, get_snappi_ports, snappi_port_selection                                             # noqa: F401
+
+import logging
+logger = logging.getLogger(__name__)
+
+pytestmark = [pytest.mark.topology('multidut-tgen')]
+
+
+def test_linecard_variation_test(snappi_api,                   # noqa: F811
```

Comment:
i've run into this problem when trying to verify this test:

```
file /var/src/sonic-mgmt_vmsvc5-t2-8800-ixia_669de75fbbfea63e8b73b319/tests/common/snappi_tests/snappi_fixtures.py, line 1225
  @pytest.fixture(scope="module")
  def get_snappi_ports(duthosts, request):
E       fixture 'get_snappi_ports_multi_dut' not found
```

could you please adapt the fix in here? https://github.com/sonic-net/sonic-mgmt/pull/14742/files#diff-b8be5d218647ad2511bf0f887565a269ded8b0150bc4c37b5bc7b0cc6c8da9e6R7

Thank you

Reply:
I have pulled in the fixes from above PR.

Thanks,
-A